src/math/warpExposure.cc

```diff
@@ -595,7 +596,8 @@ int afwMath::warpImage(
     typename DestImageT::SinglePixel padValue
 ) {
     afwGeom::Point2D const destXY0(destImage.getXY0());
-    afwMath::detail::WcsPositionFunctor const computeSrcPos(destXY0, destWcs, srcWcs);
+    afwImage::XYTransformFromWcsPair xyTransform{destWcs.clone(), srcWcs.clone()};
```

Comment:
It's disappointing that we have to clone the `Wcs`s here; `Wcs`s are sufficiently complex that I don't think we want to assume this is an efficient operation.  I think I'd prefer to change the arguments to `PTR(Wcs const)` so we can pass them through directly.  I'm ambivalent as to whether we should add a new signature and leave this deprecated for a while before removing it, or just change it now and deal with code downstream immediately.